dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthenticationRestControllerIT.java

```diff
@@ -342,6 +374,38 @@ public void testReuseTokenWithDifferentIP() throws Exception {
                 .andExpect(jsonPath("$.type", is("status")));
     }
 
+    @Test
+    public void testReuseTokenWithDifferentIPWhenIPNotStored() throws Exception {
```

Comment:
NOTE for reviewers: This test is not directly related to CSRF protection.  However, I added it to prove that (depending on your `local.cfg` settings) the JWT Token (in header or Cookie) can be reused across IP addresses.  This is a real life scenario that could be encountered if a user is behind a proxy or ISP that switches IP addresses on them during their session.  That said, this sort of behavior _only safe_ if you have CSRF protection enabled.

Assuming we merge this CSRF protection PR, we could even consider removing entirely this `jwt.login.token.include.ip` option (in a follow-up PR).  It's not really necessary if CSRF protection is there, as you are protected even if your IP changes during your session.

Reply:
I'm in favor to remove the option. In such case we should rename the test `testReuseTokenWithDifferentIPWhenIPStored` in `testReuseTokenWithoutCSRFToken`